RecoParticleFlow/PFClusterProducer/interface/PFEcalEndcapRecHitCreator.h

```diff
@@ -0,0 +1,142 @@
+#ifndef RecoParticleFlow_PFClusterProducer_PFEcalEndcapRecHitCreator_h
```

Comment:
I would leave them separate, because the mapping is done differently (in EB with the EcalTrigTowerConstituentsMap and in EE with the EcalElectronicsMapping). It can be reverted with the template, but in the future it might be useful to develop them separately because the detectors will evolve differently